core/services/job/orm.go

```diff
@@ -897,7 +916,7 @@ WHERE ocr2spec.id IS NOT NULL OR bs.id IS NOT NULL
 func (o *orm) findJob(jb *Job, col string, arg interface{}, qopts ...pg.QOpt) error {
 	q := o.q.WithOpts(qopts...)
 	err := q.Transaction(func(tx pg.Queryer) error {
-		sql := fmt.Sprintf(`SELECT * FROM jobs WHERE %s = $1 LIMIT 1`, col)
+		sql := fmt.Sprintf(`SELECT jobs.*, job_pipeline_specs.pipeline_spec_id FROM jobs JOIN job_pipeline_specs ON (jobs.id = job_pipeline_specs.job_id) WHERE jobs.%s = $1 LIMIT 1`, col)
```

Comment:
@vyzaldysanchez Are you missing an `is_primary = true` here?

Reply:
addressed, good catch.